homeassistant/components/climate/ephember.py

```diff
@@ -117,7 +117,8 @@ def device_state_attributes(self):
     @property
     def current_operation(self):
         """Return current operation ie. heat, cool, idle."""
-        mode = self._ember.get_zone_mode(self._zone_name)
+        from pyephember.pyephember import ZoneMode
+        mode = ZoneMode(self._zone['mode'])
         return self.map_mode_eph_hass(mode)
```

Comment:
For the next PR. If you call a function only once, you don't need it and can save the function frame. The map_mode_eph_hass can be cleanup.